packages/messenger-widget/src/components/Preferences/bl.tsx

```diff
@@ -0,0 +1,15 @@
+// method to open the preferences modal
+export const openPreferencesModal = () => {
+    let modal: HTMLElement = document.getElementById(
```

Comment:
const pls :)

Reply:
The value of the variable is changed in next line that is why I didn't make it const. Shall I make it to const?

Reply:
I'd use const because the value of a property within the object is changed instead the object itself.

Reply:
Ok, modified